icons/starlingbank.svg

```diff
@@ -1 +1 @@
-<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Starling Bank</title><path d="M12 0C5.3726 0 0 5.3727 0 12c0 6.627 5.3726 12 12 12 6.6279 0 12-5.373 12-12 0-6.6273-5.3721-12-12-12zm3.6094 4.7559v1.4668c-3.1771 0-5.754 2.5766-5.754 5.7539H8.3888c0-3.9882 3.2327-7.2207 7.2207-7.2207zm-1.4649 7.2207h1.4668c0 3.9889-3.233 7.2226-7.2226 7.2226v-1.4687c3.1787 0 5.7558-2.5757 5.7558-5.754Z"/></svg>
\ No newline at end of file
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Starling Bank</title><path d="M12.257.631a15.683 15.702 0 0 0-1.242.021C7.693.926 4.671 2.455 2.621 4.895A10.886 10.899 0 0 0 .397 14.903c1.132 4.11 4.681 7.326 9.071 8.223.925.188 1.572.251 2.626.247a12.232 12.247 0 0 0 6.19-1.66 11.419 11.434 0 0 0 5.636-8.102 14.876 14.895 0 0 0-.042-3.444 11.079 11.094 0 0 0-3.39-6.22 12.228 12.243 0 0 0-6.994-3.24A15.683 15.702 0 0 0 12.257.63zm3.122 3.6.005 1.283.008 1.287h-.26c-1.525.013-2.724.43-3.665 1.284a4.184 4.184 0 0 0-.995 1.287c-.332.651-.438 1.067-.498 1.915l-.05.723-1.35.004-1.359.009.02-.828c.016-.6.052-.933.118-1.241.65-2.928 3.113-5.182 6.195-5.648a9.696 9.696 0 0 1 1.117-.071zm1.455 7.75-.02.677a13.315 13.33 0 0 1-.072 1c-.458 2.964-2.794 5.329-5.92 5.982a6.88 6.888 0 0 1-1.316.117l-.874.025-.004-1.284-.008-1.283.635-.033c.721-.035 1.256-.136 1.785-.33a4.773 4.782 0 0 0 2.454-2.095c.387-.715.528-1.256.56-2.157l.021-.606 1.38-.009z"/></svg>
```

Comment:
I used `Starling Bank_Logo_Vertical.eps` from their ZIP media kit and I'm seeing a major difference. The icon looks stretched horizontally:
![image](https://user-images.githubusercontent.com/12817388/134916137-dedaa525-6949-44af-a44b-c012a4f6ae39.png)

Which file did you use to create the icon? Can you take another look at it?

Reply:
I htink I used the EPS, that's the vector one.
Maybe I converted it wrong.

Reply:
I've tried again. The icon was indeed squashed, it was 23x22 for some reason. You can check again now.